dev/com.ibm.ws.jpa_spec10_query_fat/test-applications/olgh8294/web/olgh8294Web.war/WEB-INF/classes/META-INF/persistence.xml

```diff
@@ -35,6 +35,7 @@
 
     <persistence-unit name="OLGH8294_PROPERTY_RL" transaction-type="RESOURCE_LOCAL">
         <non-jta-data-source>jdbc/JPA_NJTADS</non-jta-data-source>
+        <class>com.ibm.ws.jpa.olgh8294.model.SimpleEntityOLGH8294</class>
```

Comment:
Was this change needed for a test failure?? I wouldn't think this to be necessary for different databases

Reply:
Oh, I needed that to generate the sqlserver ddl (remember, java se environments require <class> elements, they don't scan for persistent types.  It doesn't hurt anything having it there, but I'll clean it up.

Reply:
Thanks. I appreciate cleaning this up, even though I agree there isnt any real issue having it there.

Reply:
fixed